backend/Api/Projects/ProjectController.cs

```diff
@@ -39,9 +42,76 @@ public ProjectController(ApplicationContext context, IMemoryCache cache)
                         new EngagementReadModel(e.Id, e.Name, e.State, false)).ToList()))
             .ToList();
     }
+
+    [HttpPut]
+    public ActionResult<ProjectWithConsultantsReadModel> Put([FromRoute] string orgUrlKey,
+        [FromBody] EngagementWriteModel body)
+    {
+        var service = new StorageService(_cache, _context);
+
+        var selectedOrg = _context.Organization.SingleOrDefault(org => org.UrlKey == orgUrlKey);
+        if (selectedOrg is null) return BadRequest("Selected org not found");
+
+        var customer = service.UpdateOrCreateCustomer(selectedOrg, body.CustomerName, orgUrlKey);
+        var project = service.UpdateOrCreateProject(customer, body, orgUrlKey);
+
+        var consultants = _context.Consultant.Where(c => body.ConsultantIds.Contains(c.Id)).ToList();
+        if (!consultants.Any()) return BadRequest("No consultant-ids match in db");
+
+        // Adding consultants to project
+        // TODO : What if consultants exists from before on project?
+        project.Consultants.AddRange(consultants);
+
+        var nextWeekSet = FillProjectWeekWithConsultants(consultants, project);
+
+        _context.SaveChanges();
+        _cache.Remove($"{ConsultantCacheKey}/{orgUrlKey}");
```

Comment:
Hmm. Her omgår vi jo StorageService litt, men jeg skjønner jo litt hvorfor siden vi har et relativt spesielt behov her for å populere databasen etter noen veldig spesifikke behov. Og det gir jo litt mening det også forsåvidt. 

Lurer litt på om vi kan legge hodene litt i bløtt for hva StorageService skal gjøre, og hvor grensene går. Enten så bør vi finne en måte hvor vi kan gjøre dette litt mer clean, eller så bør vi endre litt ansvarsområder her